dask/dataframe/tests/test_shuffle.py

```diff
@@ -1087,6 +1097,8 @@ def generate_raw_partd_file(compression):
     assert len(uncompressed_data) > len(compressed_data)
 
 
+# TODO does this need to be run on all shuffle methods?
+# So many parameterizations already that it adds a lot of time.
```

Comment:
In my experience, TODOs left in the code will never be actually worked on. Let's make this decision now. 

* Do you know how much time this test takes?
* The `ignore_index` and `max_branch` params feel awkward here since the test name and the comment explicitly mention the `on` parameter. Do we actually need to cross parametrize them or would it be sufficient to parameterize on one axis only, i.e. have a unique parametrized test for every variable instead of the cross product

Reply:
@rjzamora wrote this test. I also think it might be overly parameterized, but I'm not really sure what the intent is—hence the TODO. The relevant question in this PR is whether this is actually meant to test any behavior of the underlying `rearrange_by_columns_*` methods, or if it's purely testing behavior in the top-level `shuffle.shuffle` method. If so, we should remove the `shuffle_method` parametrization entirely.

_Without_ the p2p shuffle, this expands to 48 tests, and takes 15sec to run on my machine. Also, in 60% of the cases, `df_out_1._name == df_out_2._name`, which makes me wonder how relevant those cases are (they simplify to the same name, so theoretically the same graph.) Incidentally, this is breaking the p2p tests, because it's trying to reuse the same shuffle ID for multiple concurrent shuffles.

Reply:
There is also the last assert/if statement already mentioning that this is not even possible for disk so we're already running a few too many tests. the ignore_index + disk are 16? ( or is it 8? 🤔)  tests which are "useless"

I'm fine with not parametrizing over `shuffle_method`. Removing the fixture (and leaving a comment) would be fine for me.